DuckDuckGo/AppDelegate.swift

```diff
@@ -536,25 +535,14 @@ import WebKit
         importPasswordsStatusHandler.checkSyncSuccessStatus()
     }
 
-    private func stopTunnelAndShowThankYouMessagingIfNeeded() {
-        if accountManager.isUserAuthenticated {
-            return
-        }
-
-        if AppDependencyProvider.shared.vpnFeatureVisibility.isPrivacyProLaunched() && !accountManager.isUserAuthenticated {
-            Task {
-                await self.stopAndRemoveVPN(with: "subscription-check")
-            }
-        }
-    }
```

Comment:
This was a bit outdated. We no longer show "thank you" messaging, and this function was checking `isUserAuthenticated` twice. I moved this into `stopAndRemoveVPNIfNotAuthenticated` since that's really what we were doing here.